### PR TITLE
Testing combatants

### PR DESCRIPTION
**Pull request by tomaslaz**
A slight improvement on testing combatants readiness for a tournament.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,7 +13,8 @@ pelican_push:
 	docker push $(REGISTRY)/${RL_TEAM_ID}:pelican_latest
 
 pelican_test:
-	docker run $(REGISTRY)/${RL_TEAM_ID}:pelican_latest Combatant/tests/test_pelican.sh
+	PATH_TO_AGENT="data/agents/models/panther/move_north_nn"
```
Comment (nbarlowATI) [inline Makefile]: Does this line do anything, or is it an example of what the "<<PATH TO AGENT>>" should look like?

File: Makefile [hunk 2]
```diff
@@ -23,4 +24,4 @@ panther_push:
 	docker push $(REGISTRY)/${RL_TEAM_ID}:panther_latest
 
 panther_test:
-	docker run $(REGISTRY)/${RL_TEAM_ID}:panther_latest Combatant/tests/test_panther.sh
+	docker run $(REGISTRY)/${RL_TEAM_ID}:panther_latest Combatant/tests/test_panther.sh "<<PATH TO THE AGENT>>"
```
Comment (nbarlowATI) [inline Makefile]: So I guess participants need to edit the Makefile with the path to their agent here?

Would it not be possible just to have `test_pelican.py` and `test_panther.py` import AGENTS_PATH from `combatant.py` (and maybe adds "pelican" or "panther" to the end), so it automatically looks at the same agent that `load_combatant` will load?

**Review by nbarlowATI — CHANGES_REQUESTED** [review]
Looks good, though I do wonder if it might be possible to make things easier for participants by having the test_<pelican|panther>.py just use the AGENTS_PATH from combatant.py... Is there a reason this wouldn't work?

Also, it looks like the directory containing the test agents from Greg has been renamed (maybe in a previous PR? ) from `data/agents/test/panther` to `data/agents/test/panther_image_based` - I think this is misleading, as these aren't image-based (they are the latest ones that ought to work), and it also breaks the current default loading in `combatant.py`, which adds either "pelican" or "panther" to the base AGENTS_PATH...   So we should either change the logic there, and/or move the agents back...

File: Makefile (hunk 1 above)
Comment (tomaslaz) [reply Makefile]: Good catch, will remove this.

**Comment by tomaslaz** [comment]
> Also, it looks like the directory containing the test agents from Greg has been renamed (maybe in a previous PR? ) from `data/agents/test/panther` to `data/agents/test/panther_image_based` - I think this is misleading, as these aren't image-based (they are the latest ones that ought to work), and it also breaks the current default loading in `combatant.py`, which adds either "pelican" or "panther" to the base AGENTS_PATH... So we should either change the logic there, and/or move the agents back...

Nice catch! This happened as I was working with a version that had image based agents as tests and I renamed the folders. When new Greg's agent were added it went to the renamed folder.

**Comment by tomaslaz** [comment]
> Looks good, though I do wonder if it might be possible to make things easier for participants by having the test_<pelican|panther>.py just use the AGENTS_PATH from combatant.py... Is there a reason this wouldn't work?

I like this idea a lot. New changes implement this.

**Review by nbarlowATI — APPROVED** [review]
Awesome!  That was quick! :) 
Just tested `make panther_build; make panther_test` and same for pelican, and both worked perfectly! :)

**Pull request merged**